iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/TransferInstruction.java

```diff
@@ -28,9 +28,19 @@
  * </pre>
  */
 public interface TransferInstruction extends Instruction {
+
+  /**
+   * @return The list of {@link Flag} for this transfer instruction.
+   */
   List<Flag> options();
 
+  /**
+   * @return The list of sources of type {@link Argument} to transfer.
+   */
   List<Argument> srcs();
 
+  /**
+   * @return The destination of type {@link Argument} to transfer to.
+   */
```

Comment:
As a first step I changed the transfer instruction and updated javaDoc.
As the changes in transfer instruction weren't needed, I still will keep this javaDoc as it's already finished.